client/my-sites/comments/comment-list/index.jsx

```diff
@@ -93,10 +102,15 @@ export class CommentList extends Component {
 		}
 	}
 
-	changePage = page => this.setState( {
-		page,
-		selectedComments: {},
-	} );
+	changePage = page => {
+		const total = Math.ceil( ( this.props.comments.length + this.state.persistedComments.length ) / COMMENTS_PER_PAGE );
```

Comment:
Not sure we need the total here. If we're interested in that, maybe we could add it at the pager level instead.